x/ccv/provider/keeper/validator_set_update.go

```diff
@@ -13,28 +12,24 @@ import (
 	ccv "github.com/cosmos/interchain-security/v5/x/ccv/types"
 )
 
+// GetConsumerChainKey returns the store key for consumer validators of the consumer chain with `chainID`
```

Comment:
You might want to include `ConsumerValidators` in the  name of this key so it's not too generic.